osu.Game.Rulesets.Taiko/UI/TaikoPlayfieldAdjustmentContainer.cs

```diff
@@ -12,11 +13,22 @@ public partial class TaikoPlayfieldAdjustmentContainer : PlayfieldAdjustmentCont
     {
         private const float default_relative_height = TaikoPlayfield.DEFAULT_HEIGHT / 768;
 
+        public const float DEFAULT_ASPECT = 16f / 9f;
         public const float MAXIMUM_ASPECT = 16f / 9f;
         public const float MINIMUM_ASPECT = 5f / 4f;
 
         public readonly IBindable<bool> LockPlayfieldAspectRange = new BindableBool(true);
 
+        public static double AspectRatioToTimeRange(float aspectRatio)
+        {
+            const float default_time_range = 7000 / TaikoBeatmapConverter.VELOCITY_MULTIPLIER;
```

Comment:
It should be explicitly specified that such number was taken purely by visual comparison with stable. Also, can you give further insight as to why this number is being divided by `VELOCITY_MULTIPLIER`?

Reply:
Comment added in a46ca142886163eb839b4781b0fc929e5a946801

It being divided by `VELOCITY_MULTIPLIER` is to signify that it already takes into  account the 1.4x speed up of taiko. To quote the current code's comment:
```c#
// Stable internally increased the slider velocity of objects by a factor of `VELOCITY_MULTIPLIER`.
// To simulate this, we shrink the time range by that factor here.
// This, when combined with the rest of the scrolling ruleset machinery (see `MultiplierControlPoint` et al.),
// has the effect of increasing each multiplier control point's multiplier by `VELOCITY_MULTIPLIER`, ensuring parity with stable.
```

Though now that it's a constant, we could just make the default 5000 instead. Or we could include this comment here. Do let me know if one of these would be preferable